Gopkg.toml

```diff
@@ -9,8 +9,34 @@ required = [
   "k8s.io/code-generator/cmd/informer-gen",
   "k8s.io/code-generator/cmd/openapi-gen",
   "k8s.io/gengo/args",
+  "github.com/kevinburke/go-bindata",
 ]
 
+[prune]
+  go-tests = true
+  non-go = true
+  unused-packages = true
+
+[[prune.project]]
```

Comment:
Don't this need to be indent under `[prune]` section? (https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md#prune)

Reply:
Indentation isn't significant in TOML.